sdcm/cluster_k8s/__init__.py

```diff
@@ -2593,10 +2593,6 @@ def connection_bundle_file(self) -> Path | None:
     def node_setup(self, node: BaseScyllaPodContainer, verbose: bool = False, timeout: int = 3600):
         if self.test_config.BACKTRACE_DECODING:
             node.install_scylla_debuginfo()
-
-        if self.test_config.MULTI_REGION:
-            node.datacenter_setup(self.datacenter)  # pylint: disable=no-member
```

Comment:
removing this might be problematic, for non AWS cases.
we should test some GCP multi-region case to validate.

Reply:
This is `k8s` file.